graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/alarms/AlertCondition.java

```diff
@@ -56,13 +45,13 @@
 
     Map<String, Object> getParameters();
 
-    Integer getBacklog();
+    Integer getBacklogSize();
```

Comment:
This is a breaking change which affects all existing alarm callbacks (and there are actually some plugins providing those), so I would not rename that method in the Graylog 1.x version line.